azurelinuxagent/daemon/resourcedisk/default.py

```diff
@@ -339,14 +339,14 @@ def mkfile(self, filename, nbytes):
                 # Probable errors:
                 #  - OSError: Seen on Cygwin, libc notimpl?
                 #  - AttributeError: What if someone runs this under...
-                fd = None
+                fd = None # pylint: disable=C0103
 
                 try:
-                    fd = os.open(
+                    fd = os.open( # pylint: disable=C0103
                         filename,
                         os.O_CREAT | os.O_WRONLY | os.O_EXCL,
                         stat.S_IRUSR | stat.S_IWUSR)
-                    os.posix_fallocate(fd, 0, nbytes)
+                    os.posix_fallocate(fd, 0, nbytes) # pylint: disable=no-member
```

Comment:
so one can disable by code (C0103) or name (disable-no-member)?

Reply:
Yes, that's correct.